Seneca-CDOT/telescope
# [Chore] Remove firebase and users microservice

## Description

Since we're going to use `supabase` as telescope's main database, there's no need for a `firebase` emulator in development.
This PR removes all references to the emulator and the `users` service.



## Steps to test the PR
- Run `docker-compose --env-file ./config/env.development up --build`
- Make sure `telescope` runs as expected

## Checklist
